socorro/schemas/telemetry_socorro_crash.json

```diff
@@ -0,0 +1,1122 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
I changed the indentation to 2 spaces and renamed the file to something more specific. Otherwise, it's identical.